kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/ProcessResource.java

```diff
@@ -328,9 +329,12 @@ public Response startProcessWithCorrelationKeyFromNodeIds(@javax.ws.rs.core.Cont
         } catch (SecurityException e) {
             return forbidden(errorMessage(e, e.getMessage()), v, conversationIdHeader);
         } catch (Exception e) {
+            if (checkExceptionType(e)) {
+                return badRequest(errorMessage(e, e.getMessage()), v, conversationIdHeader);
```

Comment:
Shouldn't you return rather HTTP 409 instead of 400?

Reply:
done